### DISCUSSION TITLE
Fill in the grid

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#998] [Galaxy Trivia] [answered]
The number of Xs in each row and column should match the corresponding numbers for each row and column.

|  | 3 | 4 | 1 | 3 | 2 |
|---|---|---|---|---|---|
| **5** | x | x | x | | |
| **1** |  | x |  |  | |
| **2** | x |  |  |  | | 
| **3** |  | | | | | 
| **2** |  |  |  |  | |

**Comment by tomerf** [comment, marked as answer]
| | 3 | 4 | 1 | 3 | 2 |
|---|---|---|---|---|---|
| **5** | x | x | x | x | x |
| **1**| | x |  |  |
| **2** | x | x | |  |  |
| **3** | x | x |  | x |  |
| **2** |  |  |  | x | x |

**Reply by tomerf** [reply]
There's more than one solution, it's possible to switch 2 columns between the last 2 lines (one column where x is in the first line and one column where the x is in the second line) and still get a valid solution.
